presto-main/src/main/java/com/facebook/presto/operator/scalar/IpPrefixFunctions.java

```diff
@@ -137,6 +144,222 @@ public static boolean isPrefixSubnetOf(@SqlType(StandardTypes.IPPREFIX) Slice fi
         return between(ipSubnetMin(second), ipSubnetMin(first), ipSubnetMax(first)) && between(ipSubnetMax(second), ipSubnetMin(first), ipSubnetMax(first));
     }
 
+    @Description("Combines the input set of IP prefixes into the smallest contiguous CIDR ranges possible.")
```

Comment:
Is this smallest or fewest? That is, is it returning a single smallest range that covers everything (smallest range)  or a list of ranges that covers everything (fewest ranges)?

Reply:
Agreed -- "fewest ranges" is better.